examples/todo/modules/shared-data/src/main/scala/com/my/shared_data/lib/OrderedOps.scala

```diff
@@ -0,0 +1,24 @@
+package com.my.shared_data.lib
+
+import com.my.shared_data.schema.Updates._
+
+object OrderedOps {
+
+  object implicits {
+
+    implicit val todoUpdateOrdering: Ordering[TodoUpdate] =
+      new Ordering[TodoUpdate] {
+
+        def compare(x: TodoUpdate, y: TodoUpdate): Int = {
+          def order(todo: TodoUpdate): Int = todo match {
+            case _: CreateTask   => 0
+            case _: ModifyTask   => 1
+            case _: CompleteTask => 2
+            case _: RemoveTask   => 3
+          }
+
+          order(x) compareTo order(y)
+        }
+      }
```

Comment:
So, two todos are equally the same if they are of the same type?

Reply:
I was just aiming for grouping all like Tasks together when applying them onto the DataState. I was not concerned with ordering between like Tasks

Reply:
Ok, but it's a default implicit so it may arise additional errors with sorting/ordering (theoretically, didn't check). Take a look how cats derive `Eq` from `Order`.

We encountered many times in the past that for the ordering like that:
```
case Snapshot(ordinal: Long, a: Foo, b: Bar, c: Baz)

object Snapshot {
  implicit val order: Order[Snapshot] = ... a.ordinal > b.ordinal // (or sth like that, based on one field only)
}
```

snapshots with the same ordinal would be considered equal, even though `a`, `b` and `c` properties are different.